code/modules/mob/living/carbon/human/emote_vr.dm

```diff
@@ -8,9 +8,15 @@
 			message = "awoos loudly. AwoooOOOOoooo!"
 			m_type = 2
 
+		if ("flip")
+			if(!src.sleeping || !src.resting || !src.buckled || !src.restrained())
+				src.SpinAnimation(7,1)
+			m_type = 1
```

Comment:
I think m_type 1 is silent and m_type 2 is noisy? So this is right. Though I'm sure flipping makes some kinda noise... it's probably not one a blind person could tell what it was.